src/meson.build

```diff
@@ -680,7 +680,7 @@ if get_option('tests').enabled()
     test(name, find_program(name + '.py'),
       env: env,
       depends: name == 'check-symbols' ? defs_list : [],
-      suite: ['src'],
+      suite: ['src'] + (name == 'check-static-inits' ? ['slow'] : []),
```

Comment:
Surprised about this one.

Reply:
Marked ones takes longer than 1s and this ones takes 2s, perhaps can be optimized somehow, yet, marking as slow doesn't mean to skip the test.

Reply:
Marked ones takes longer than 1s and this ones takes 2s, perhaps can be optimized somehow, yet, marking as slow doesn't mean to skip the test.

Reply:
> Marked ones takes longer than 1s and this ones takes 2s, perhaps can be optimized somehow, yet, marking as slow doesn't mean to skip the test.

Okay yeah if 1s is the limit I can see that. Nothing to optimize, it's calling `readelf` or equivalent.